go/vrf/conversion_test.go

```diff
@@ -58,8 +58,8 @@ func TestSEG1EncodeDecode(t *testing.T) {
 		t.Fatal(err)
```

Comment:
Function name changed, so the test name probably should, too.

Reply:
done